jest.config.mjs

```diff
@@ -0,0 +1,11 @@
+// export default
```

Comment:
`export default config;` in the end looks pretty explicit; remove comment maybe?